index.html

```diff
@@ -0,0 +1,383 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="wrapper">
+      <nav class="container">
+        <div id="logo">
+          <a href="#"><img src="./assets/img/logo.png" alt="logo" /></a>
+        </div>
+        <div class="toggle">
+          <a href="#"><img src="./assets/img/toggle.png" alt="toggle" /></a>
+        </div>
+      </nav>
+    </header>
+    <section class="showcase">
+      <div class="showcase_text">
+        <h1 id="tagline">
+          Biftu <br />
+          Girma
+        </h1>
+        <p>
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look at my work. <br /><br />
+          If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+
+        <a class="primary-btn" href="#">Start collaboration</a>
+      </div>
+
+      <div class="showcase_img">
+        <img
+          src="./assets/img/headline_img.png"
+          alt="headline_img"
+          class="headline_img"
+        />
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="work-card">
+        <h2 id="tagline">Projects</h2>
+        <img src="./assets/img/laptop.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
```

Comment:
- [ ] Kindly wrap the technologies in `ul` tag 👍 and each of them a `li`. This way will be semantically correct and the screen readers